architecture.md

```diff
@@ -13,14 +13,24 @@ The MongoDB Community Kubernetes Operator is a [Custom Resource Definition](http
 You create and update MongoDB resources by defining a MongoDB resource definition. When you apply the MongoDB resource definition to your Kubernetes environment, the Operator:
 
 1. Creates a [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) that contains one [pod](https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/) for each [replica set](https://docs.mongodb.com/manual/replication/) member. 
-1. Creates two [containers](https://kubernetes.io/docs/concepts/containers/overview/) in each pod:
+1. Writes the Automation configuration as a [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/) and mounts it to each pod. 
+1. Creates one [init container](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/) and two [containers](https://kubernetes.io/docs/concepts/containers/overview/) in each pod:
+
+   - An init container which copies the `cmd/prestop` binary to the main `mongod` container. </br>
```

Comment:
what's the `</br>` doing here?

Reply:
There's a break for every item in the containers list since before. Should we remove that?

Reply:
Can you check if it is necessary? if not needed, please remove 👍

Reply:
I think it's fine without them, removed.